targets/PythonSdk/templates/API.py.ejs

```diff
@@ -0,0 +1,34 @@
+import json
+import PlayFabErrors
+import PlayFabHTTP
+import PlayFabSettings
+
+<%- generateApiSummary("", api, "description")%>
+<% for(var i in api.calls) { var apiCall = api.calls[i];
+%><%- generateApiSummary("", apiCall, "summary")
+%><%- getDeprecationAttribute("", apiCall)
+%>def <%- apiCall.name %>(request, callback, customData = None, extraHeaders = None):
+<%- getRequestActions("    ", apiCall) %>
+    def wrappedCallback(playFabResult, error):
+<%- getResultActions("        ", apiCall, api) %>
+        callback(playFabResult, error)
+
+    PlayFabHTTP.DoPost("<%- apiCall.url %>", request, <%- getAuthParams(apiCall) %>, extraHeaders, wrappedCallback)
+
+<% } %><% if (hasClientOptions) {
+%># Determine if the client session ticket is set, without actually making it public
+def IsClientLoggedIn():
```

Comment:
I think these hasClientOptions functions should be defined at the top of this file, before the API calls.